src/uvc_ros_driver.cpp

```diff
@@ -835,9 +835,7 @@ inline void uvcROSDriver::deinterleave(const uint8_t *mixed, uint8_t *array1,
 		i += 16;
 		mixed += 32;
 	}
-
-#endif
-
+#else
```

Comment:
Could you comment on this change?

Reply:
@LorenzMeier I guess this loops are mutual exclusive based on the architecture, so I guess this change makes sense.

Reply:
Exactly. Otherwise you have to comment out the section which is now between #else and #endif.